facia/app/utils/FaciaPicker.scala

```diff
@@ -8,24 +8,39 @@ import play.api.mvc.RequestHeader
 
 class FaciaPicker extends GuLogging {
 
-  // we don't yet use these parameters but they will play into dcrCouldRender later on
+  val SUPPORTED_COLLECTIONS: Set[String] =
+    Set("dynamic/fast", "dynamic/slow", "fixed/small/slow-IV", "fixed/large/slow-XIV")
```

Comment:
I took the supported containers from here: https://github.com/guardian/dotcom-rendering/issues/4720. Is my understanding correct that these are supported by DCR at the moment?

Reply:
I think that's a good list yes. You can also look at [this file](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/lib/DecideContainer.tsx) which is perhaps a better source for this information

Reply:
Can you please add these links inside the code directly for future travellers?

Reply:
Yes, good point!